### PR TITLE
Clean up variable scopes

### PR DESCRIPTION
**Pull request by waltdisgrace**
This is a preliminary PR that will address #1708.

### CODE REVIEW

File: tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py [hunk 1]
```diff
@@ -44,11 +44,11 @@ def setUp(self):
         super().setUp()
         self._proxy = get_object(TOP_OBJECT)
         self._devs = _DEVICE_STRATEGY()
-        ((_, (poolpath, _)), _, _) = Manager.Methods.CreatePool(
+        ((_, (self._poolpath, _)), _, _) = Manager.Methods.CreatePool(
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py]: This should stay local. It's not being used in any of the tests.

File: tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py [hunk 2]
```diff
@@ -44,11 +44,11 @@ def setUp(self):
...
             self._proxy,
             {"name": self._POOLNAME, "redundancy": (True, 0), "devices": self._devs},
         )
-        self._pool_object = get_object(poolpath)
+        self._pool_object = get_object(self._poolpath)
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py]: This should also be local.

File: tests/client-dbus/tests/dbus/pool/test_create_filesystem.py [hunk 3]
```diff
@@ -87,8 +88,8 @@ def testDuplicateSpecs(self):
         self.assertEqual(len(result), 1)
         self.assertEqual(rc, StratisdErrors.OK)
 
-        (_, fs_name) = result[0]
-        self.assertEqual(fs_name, new_name)
+        (_, self._FSNAME) = result[0]
```
Comment (mulkieran) [inline tests/client-dbus/tests/dbus/pool/test_create_filesystem.py]: pylint will always be disconcerted to find an assignment to a class-level variable within a test method. Remember that we generally only use identifiers in all caps for constants, and if you are assigning to such an identifier within a method, you're definitely not treating it as a constant. If you look at the scope and the user of the identifer, it is completely local to the method, so you should change back here to the way it was before, and similarly in the method below.

File: tests/client-dbus/tests/dbus/pool/test_create_filesystem.py [hunk 4]
```diff
@@ -38,17 +38,18 @@ class CreateFSTestCase(SimTestCase):
...
         Start the stratisd daemon with the simulator.
         """
         super().setUp()
         self._proxy = get_object(TOP_OBJECT)
-        self._devs = _DEVICE_STRATEGY()
+        devs = _DEVICE_STRATEGY()
         ((_, (poolpath, _)), _, _) = Manager.Methods.CreatePool(
             self._proxy,
-            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": self._devs},
+            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": devs},
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_create_filesystem.py]: Given that this is local, do the same thing as [here](https://github.com/stratis-storage/stratisd/pull/1710/files#diff-1de2f36d829f7f87ccc8647b0450d353R84).

File: tests/client-dbus/tests/dbus/pool/test_create_filesystem.py [hunk 5]
```diff
@@ -102,21 +103,21 @@ class CreateFSTestCase1(SimTestCase):
...
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_create_filesystem.py]: Given that this is local, do the same thing as [here](https://github.com/stratis-storage/stratisd/pull/1710/files#diff-1de2f36d829f7f87ccc8647b0450d353R84).

File: tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py [hunk 6]
```diff
@@ -43,10 +43,10 @@ def setUp(self):
         """
         super().setUp()
         self._proxy = get_object(TOP_OBJECT)
-        self._devs = _DEVICE_STRATEGY()
+        devs = _DEVICE_STRATEGY()
         ((_, (poolpath, _)), _, _) = Manager.Methods.CreatePool(
             self._proxy,
-            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": self._devs},
+            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": devs},
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py]: Given that this is local, do the same thing as [here](https://github.com/stratis-storage/stratisd/pull/1710/files#diff-1de2f36d829f7f87ccc8647b0450d353R84).

File: tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py [hunk 7]
```diff
@@ -92,22 +92,22 @@ class DestroyFSTestCase1(SimTestCase):
...
         """
         super().setUp()
         self._proxy = get_object(TOP_OBJECT)
-        self._devs = _DEVICE_STRATEGY()
-        ((_, (self._poolpath, _)), _, _) = Manager.Methods.CreatePool(
+        devs = _DEVICE_STRATEGY()
+        ((_, (poolpath, _)), _, _) = Manager.Methods.CreatePool(
             self._proxy,
-            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": self._devs},
+            {"name": self._POOLNAME, "redundancy": (True, 0), "devices": devs},
```
Comment (jbaublitz) [inline tests/client-dbus/tests/dbus/pool/test_destroy_filesystem.py]: Given that this is local, do the same thing as [here](https://github.com/stratis-storage/stratisd/pull/1710/files#diff-1de2f36d829f7f87ccc8647b0450d353R84).

**Comment by waltdisgrace** [comment]
Rebased.

**Comment by jbaublitz** [comment]
I'm going to take a look at the branch in full.

**Comment by mulkieran** [comment]
Superceded by #1721, so closing.

**Pull request closed without merging**